pkg/audit/manager_test.go

```diff
@@ -1,11 +1,66 @@
 package audit
 
 import (
+	"os"
+	"reflect"
 	"testing"
 
 	"github.com/pkg/errors"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
 )
 
+func Test_newNSCache(t *testing.T) {
+	tests := []struct {
+		name string
+		want *nsCache
+	}{
+		{
+			name: "test",
+			want: &nsCache{
+				cache: map[string]corev1.Namespace{},
+			},
+		},
+	}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			if got := newNSCache(); !reflect.DeepEqual(got, tt.want) {
+				t.Errorf("newNSCache() = %v, want %v", got, tt.want)
+			}
+		})
+	}
+}
+
+func Test_truncateString(t *testing.T) {
+	// Test case 1
```

Comment:
Wrapping these test cases in individual calls to `t.Run()` would associate the test case with the error in `go test` output.

Reply:
updated